test/testplan/testplan.md

```diff
@@ -0,0 +1,361 @@
+# Web Monetization Extension Test Plan
+
+## Introduction
+
+The purpose of this document is to guide end-user testing of the Web Monetization browser extension.
+
+## Audience
+
+Any person interested in testing the beta release of the Web Monetization extension.  
+There are no requirements for technical competencies.
+
+We focus on functional test cases, and the goal is to record and report the actual behavior that was observed when it differs from the expected behavior.
+
+## Strategy
+
+One of the strategic goals of the Interledger Foundation is to grow Web Monetization awareness and adoption such that there are 1,000 web monetized websites and 100,000 active Web Monetization users (by 31 December 2024).
+
+One of the long-term goals is to grow Web Monetization adoption by a factor of 10, in the next 5 years (by December 2029) such that Web Monetization functionality is either native to the most widely used Web browsers, or readily available through browser extensions on Web browser stores and marketplaces.
+
+## Objectives
+
+1. Validating that, once installed, users can configure Web Monetization settings on the extension.
+2. Validating that once the extension is successfully connected to a digital wallet, a user is able to.
+3. For web monetized websites, verify that the monetization event gets triggered.
+
+## Dependencies
+
+### Digital Wallets
+
+Web Monetization-enabled digital wallets are provided by licensed service providers who are regulated by the laws of the countries in which they operate. They provide digital wallets that can send or receive Web Monetization payments.
+
+Wallet availability in a specific country, or availability in particular currency depends on the wallet provider.
+To learn more, visit a specific wallet provider’s website. Below are the available wallet providers:
+
+- [Fynbos](https://wallet.fynbos.app/wallet)
+- [GateHub](https://gatehub.net/mobile)
+
+## Testing
+
+### Tools and Resources
+
+The resources and tools that you will need to test Web Monetization depend on your point of view.
+The Web Monetization personas have different perspectives and needs for testing:
+
+Persona 2: A website owner that wants to receive WM payments.
+
+| People                                                                              | Hardware                                                          | Digital wallet                                                                  | Environment & software                                                                        |
```

Comment:
Suggestions: Column title "Persona". 

Then instead of Persona 1/2 in the other cells, maybe:
"Website Visitor: A visitor who wants to pay..."
"Website Owner: A site owner that wants to..."

Reply:
Cool and thanks. Have used "Role" instead of "Persona"